python/mxnet/contrib/onnx/mx2onnx/export_onnx.py

```diff
@@ -294,7 +298,7 @@ def create_onnx_graph_proto(self, sym, params, in_shape, in_type, verbose=False)
                     # If converted node is NodeProto, add it in processed nodes list
                     elif isinstance(converted_node, NodeProto):
                         onnx_processed_nodes.append(converted_node)
-                        if idx == (len(mx_graph) - 1):
+                        if converted_node.name in output_names:
```

Comment:
This needs modification for cases when there is no converted_node.name - working on this.